requirements.txt

```diff
@@ -3,3 +3,5 @@ numpy==1.17.2
 matplotlib==3.1.1
 scikit-learn==0.21.3
 pandas==0.25.1
+fpdf==1.7.2
```

Comment:
Why both are used?

Reply:
the PDF dependencies have been fixed and only fpdf2 used